res/controllers/Denon-MC7000.midi.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version='1.0' encoding='utf-8'?>
-<MixxxControllerPreset mixxxVersion="2.3" schemaVersion="1">
+<MixxxControllerPreset mixxxVersion="2.3.1" schemaVersion="1">
     <info>
     	<name>Denon MC7000</name>
     	<author>OsZ</author>
-    	<description>Denon MC7000 mapping. Check your Linux Kernel version to get the Audio Interface working - see WIKI</description>
+    	<description>Denon MC7000 mapping. Check your Linux Kernel version to get the Audio Interface working - Details in user manual</description>
```

Comment:
Denon MC700 mapping is redundant, that is already in the name. Also, not everyone is a linux user :D

```suggestion
    	<description>A professional 4-channel DJ controller featuring 2 separate audio interfaces. Please refer to the Mixxx user manual to get the audio interfaces working on Linux.</description>
```

Reply:
Nice suggestion. I changed it accordingly.